DNN Platform/Library/Common/Globals.cs

```diff
@@ -3077,158 +3164,168 @@ public static string DateToString(DateTime DateValue)
         /// <summary>
         /// Gets the hash value.
         /// </summary>
-        /// <param name="HashObject">The hash object.</param>
-        /// <param name="DefaultValue">The default value.</param>
+        /// <param name="hashObject">The hash object.</param>
+        /// <param name="defaultValue">The default value.</param>
         /// <returns>HashOject's value or DefaultValue if HashObject is null.</returns>
-        public static string GetHashValue(object HashObject, string DefaultValue)
+        [Obsolete("Deprecated in v9.8.1, scheduled removal in v10")]

```

Comment:
Why is this being removed in v10 instead of v11?

Reply:
Addressed in latest commit, thanks.